package-lock.json

```diff
@@ -1621,7 +1621,7 @@
       "dev": true
     },
     "node_modules/@types/node": {
-      "version": "20.0.0",
+      "version": "^20.0.0",
```

Comment:
This generates the error when `npm install` because it is a invalid format.
```
44 verbose stack TypeError: Invalid Version: ^20.0.0
44 verbose stack     at new SemVer (/usr/local/lib/node_modules/npm/node_modules/semver/classes/semver.js:38:13)
44 verbose stack     at compare (/usr/local/lib/node_modules/npm/node_modules/semver/functions/compare.js:3:32)
44 verbose stack     at Object.gte (/usr/local/lib/node_modules/npm/node_modules/semver/functions/gte.js:2:30)
44 verbose stack     at CanPlaceDep.checkCanPlaceCurrent (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/can-place-dep.js:173:51)
44 verbose stack     at CanPlaceDep.checkCanPlace (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/can-place-dep.js:157:27)
44 verbose stack     at new CanPlaceDep (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/can-place-dep.js:114:26)
44 verbose stack     at PlaceDep.place (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:123:19)
44 verbose stack     at new PlaceDep (/usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/place-dep.js:73:10)
44 verbose stack     at /usr/local/lib/node_modules/npm/node_modules/@npmcli/arborist/lib/arborist/build-ideal-tree.js:990:31
44 verbose stack     at Array.map (<anonymous>)
```

`package-lock.json` should only be modified by npm commands.

Also, instead you should use the newest version, [20.11.16](https://www.npmjs.com/package/@types/node/v/20.11.16).

Reply:
You should be able to add that version to the project by running:

```
$ npm install --save-dev @types/node@^20.11.16
```